web/helpers/validations.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Community.Validations do
+  import Ecto.Changeset
+
+  def validate_url_format(changeset, field) do
+    format = ~r/http(s)?.*/
```

Comment:
I don't know how I feel about that. I like that right now, this is completely self contained. You could move this anywhere else in the code and it makes sense. Moving it externally when it is not being used elsewhere feels strange.